CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.3 (2018-11-20)
+
+- Fixed colorpicker component firing the `valueChanges` event too many times when a form group is updated. [blackbaud/skyux-colorpicker#5](https://github.com/blackbaud/skyux-colorpicker/pull/5)
```

Comment:
should that say "from firing" ?

Reply:
Uh. That probably make more sense.  One second

Reply:
"Fixed (thing) to..."

Reply:
@Blackbaud-SteveBrush check the updated string. (The above still shows the old one)

`Fixed colorpicker component to stop firing the `valueChanges` event multiples times when a form group is updated. `